lxc.md

```diff
@@ -123,7 +123,6 @@ Set up hostfiles on each box with hardcoded IP addresses
 
 
 ```
-127.0.0.1 localhost n1 n1.local
```

Comment:
Yicheng Qin from etcd says [1] a possible reason may be that listen-peer-urls is at localhost:2380 by default, which cannot receive message from 192.168.122.12:2380.

[1] https://groups.google.com/d/msg/etcd-dev/AslsLq2Lddk/83L_3jBcfNUJ